README.md

```diff
@@ -201,24 +201,28 @@ pip install 'polars[numpy,pandas,pyarrow]'
 | Tag        | Description                                                                  |
 | ---------- | ---------------------------------------------------------------------------- |
 | **all**    | Install all optional dependencies (all of the following)                     |
-| pandas     | Install with pandas for converting data to and from pandas DataFrames/Series |
```

Comment:
It makes more sense to have pandas / numpy / pyarrow on top, as those are most relevant. In fact we can just remove the rest (or this entirely) and point to the user guide.

Reply:
Yup, I like dropping the rest of the table - maybe we just direct the reader to `pl.show_versions()` if they want to see the optional deps.